tests/letstest/README.md

```diff
@@ -15,30 +15,60 @@ simple aws testfarm scripts for certbot client testing
     are needed, they need to be requested via online webform.
 
 ## Usage
+  - To install the necessary dependencies, run:
+```
+sudo apt install awscli python-yaml python-boto3 fabric
+```
+
   - Requires AWS IAM secrets to be set up with aws cli
   - Requires an AWS associated keyfile <keyname>.pem
 
 ```
->aws configure --profile HappyHacker
+>aws configure --profile <profile name>
 [interactive: enter secrets for IAM role]
->aws ec2 create-key-pair --profile HappyHacker --key-name MyKeyPair --query 'KeyMaterial' --output text > MyKeyPair.pem
+>aws ec2 create-key-pair --profile <profile name> --key-name <key name> --query 'KeyMaterial' --output text > whatever/path/you/want.pem
 ```
+Note: you can pick any file name for `<key-name>`, but it must have a `.pem` extension. Also, whatever you pick for `<profle name>` will be shown
```

Comment:
Are you sure about the need for the `.pem` extension? Unless the output of `aws ec2 describe-key-pairs` automagically deletes it, none of our current keys have this extension in their name.

Reply:
Based on the [official docs](https://docs.aws.amazon.com/cli/latest/reference/ec2/create-key-pair.html), no, but the setup email I got said: "and <key name> can be whatever you want, but *must* have a .pem extension". I think the email may be out of date? I'll update this.